.github/workflows/docker.yml

```
@@ -39,6 +39,6 @@ jobs:
             --rm \
             -v "$(pwd):/app" -w /app \
             metamask-mobile-builder:latest \
-            bash -c 'yarn && yarn setup --build-ios'
+            bash -c 'yarn && yarn setup --build-ios --no-install-pods'
```

It seems that the `pod:install` step was failing already. I'm not sure why, or when the failure started, because previously the failure was suppressed.

I've added this flag to skip the pod install step so that this pre-existing failure doesn't block this improvement.